LDK/test/src/org/labkey/test/util/external/labModules/LabModuleHelper.java

```diff
@@ -426,4 +439,50 @@ public void initiatePipelineJob(String importAction, List<String> files, String
 
         _test._fileBrowserHelper.selectImportDataAction(importAction);
     }
+
+    public void defineViralAssayWithAdditionalFields(String provider, String label, Map<String, String> batchFields, Map<String, String> runFields, Map<String, String> resultFields)
```

Comment:
Why is this in the helper class, and not in the actual test (like ViralLoadAssayTest or whatever uses this)

Reply:
Not too sure, @labkey-jeckels any idea?
Looks like this was part of 17.2 merges to wnprc18.3 branch.

Reply:
Now moved.